runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -79,6 +80,19 @@ String getTagString(int itemId, boolean variation)
 		return config;
 	}
 
+	public String getQuantityOfItemForTag(String tag, int itemId, boolean variation)
+	{
+		itemId = getItemId(itemId, variation);
+
+		String config = configManager.getConfiguration(CONFIG_GROUP, ITEM_QUANTITY_KEY_PREFIX + itemId + "." + tag);
+		if (config == null)
+		{
+			return "";
+		}
+
+		return config.trim();
```

Comment:
I don't think trimming is necessary, but if you need to do it when you parse--do it there not here.